testing/upload_validation.ts

```diff
@@ -1194,43 +1211,52 @@ const legacyPackMetadataSchema = validateFormulas(
         "Specify the domain that your pack makes http requests to using `networkDomains: ['example.com']`",
       path: ['networkDomains'],
     },
-  )
-  .superRefine((untypedData, context) => {
+  );
+const packMetadataSchemaBySdkVersion = [
```

Comment:
Mind giving this a static type?

Right now if somebody adds a new entry to the list and gets the interface wrong, TS will just happily infer this as the union of whatever it was given and we'll fail at runtime instead.

Reply:
Done